__mocks__/@react-native-firebase/crashlytics.js

```diff
@@ -0,0 +1 @@
+export default {};
```

Comment:
Would it be OK if we also make a requirement that any mock file also should have an explanation for why it's mocked? I think that would be super helpful.

Reply:
I think that sounds great. The answer won't always be the same so it is good context to add for sure and also will help people understand all the reasons why something could or should be mocked.

Reply:
But also would say we can do that in a cleanup step.